### ISSUE TITLE
Application crash when click on link in iframe

### ISSUE THREAD

**Issue by Bill-Niz** [nwjs/nw.js#7181]
NWJS Version :v0.41.2
Node : v12.9.1
Chromium : 77.0.3865.9
Operating System : macOS 10.15

### Expected behavior

When clicking on link with `target=_blank`,  the url should open in external browser.

### Actual behavior

The application crash

### How to reproduce

/!\ **The this _iframe.html_ must be hosted.** /!\ 

## index.js

```javascript
const GUI = require('nw.gui')

nw.Window.open('index.html', {
  width: 900,
    height: 800,
}, function(win) {
   win.on('new-win-policy', (frame, url, policy) => {
        policy.ignore()
        GUI.Shell.openExternal(url)
      })
});
```

## index.html

```html
<!DOCTYPE html>
<html>

<head>
    <title>Iframe</title>
</head>
<body>
    <div>
        <a href="https://stackoverflow.com/"
            target="_blank" rel="external">This wont crash </a>
            <!-- Iframe.html must be hosted !!!-->
        <iframe src="http://127.0.0.1:5501/iframe.html" style="width: 100%; height: 200px;">
        </iframe>
    </div>
</body>

</html>
```

## iframe.html

```html
<!DOCTYPE html>
<html>
  <head>
    <title>iframe</title>
  </head>
  <body>
    <h1>IFRAME</h1>
    <a href="https://github.com" target="_blank" rel="external">this will crash in iframe</a>
  </body>
</html>
```

## Stack trace
```
Received signal 11 SEGV_MAPERR 000000000000
 [0x000106bc68b9]
 [0x000106ae0633]
 [0x000106bc67d1]
 [0x7fff71dd0b1d]
 [0x7fff71d8eabe]
 [0x00010bcba323]
 [0x00010a9da1bf]
 [0x00010b0291cb]
 [0x00010ac1652a]
 [0x00010ac15857]
 [0x00010a886c79]
 [0x00010a8867a3]
 [0x00010a9c37e7]
 [0x00010a885b95]
 [0x00010acd5709]
 [0x00010acd5b31]
 [0x00010acdb80a]
 [0x00010accb713]
 [0x00010b035990]
 [0x00010aaf8c53]
 [0x00010b03564b]
 [0x00010aaf752e]
 [0x00010bc18b35]
 [0x00010bd14e12]
 [0x00010bc1592b]
 [0x00010bc14d4f]
 [0x000106b56d8f]
 [0x000106b6cb21]
 [0x000106b6d049]
 [0x00010bbe278a]
 [0x000106af68ea]
 [0x000106b04aff]
 [0x7fff3a8eda21]
 [0x7fff3a8ed9c0]
 [0x7fff3a8d0f84]
 [0x7fff3a8d0590]
 [0x7fff3a8cfe13]
 [0x7fff3cf6ccad]
 [0x00010bbe2718]
 [0x0001120bfe75]
 [0x0001120b02d9]
 [0x0001060db41b]
 [0x00011213fdef]
 [0x00010bbe2d83]
 [0x000106b04468]
 [0x000106b6d438]
 [0x000106b34607]
 [0x00010bd26bce]
 [0x000106434446]
 [0x0001093a0aab]
 [0x000106433694]
 [0x000103321d9b]
 [0x00010326c6e0]
 [0x7fff71bcf405]
[end of stack trace]
```

**Comment by rogerwang** [comment]
Thanks for reporting. I'm trying to reproduce your issue. How did you load index.js?

**Comment by Bill-Niz** [comment]
@rogerwang 
`/Applications/nwjs.app/Contents/MacOS/nwjs . --remote-debugging-port=9222`

**Comment by rogerwang** [comment]
Could you elaborate? I don't see where index.js is referred.

**Comment by Bill-Niz** [comment]
You can just clone https://github.com/Bill-Niz/nwiframe

1. Host` iframe.html` in local
2. Edit the **_iframe src_**  in `index.html` to match the url hosting of `iframe.html`
3. run nodewebkit in root directory

**Comment by CosmoMyzrailGorynych** [comment]
I experienced it with nw v0.41.2, but it seems to be fixed in nw v0.42.4

**Comment by Bill-Niz** [comment]
Fixed in nw v0.42.5
